.vscode/tasks.json

```diff
@@ -30,6 +30,17 @@
 			"problemMatcher": [
 				"$tsc-watch"
 			]
-        }
+        },
+		{
+			"label": "OpenCoverageTestReport",
+			"type": "shell",
+			"command": "./coverage/index.html",
```

Comment:
I tested it on Mac. It will fail with "Permission Denied" error, either from here or by typing this relative file path in command line:
![image](https://user-images.githubusercontent.com/57521938/112718107-d7186d00-8f2b-11eb-8269-ebf8ec5959ac.png)

"open ./coverage/index.html" works from Mac command line, but it doesn't work on Windows.

Below is a solution:
`		{
			"label": "OpenCoverageTestReport",
			"type": "shell",
			"windows": {
				"command": "open ./coverage/index.html"
			},
			"osx" : {
				"command": "open ./coverage/index.html"
			},
			"dependsOn": "OpenMochawesomeTestReport"
		},
`

Reply:
Thanks Sen, that worked perfectly for me and has been updated. I'll be sure to go check out that bug PR. Kind of glad it is a bug and its not just another scenario where a traditional Windows user is completely lost on a Mac. lol